### PR TITLE
REL-3456: Adding Zorro to ITAC

### PR DESCRIPTION
**Pull request by sraaphorst**
This is the code to add the instrument, Zorro (a permanent visitor instrument at GS replacing DSSI), to ITAC. This is mainly a call for comments: I still have to figure out how to do some testing with ITAC in order to verify that this works properly, although it is a straightforward modification of the code implemented to add ʻAlopeke to GN to replace the GN DSSI.

### CODE REVIEW

File: phase1-data/src/main/java/edu/gemini/tac/persistence/phase1/blueprint/zorro/ZorroBlueprint.scala [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  override def getDisplayOther = BlueprintBase.DISPLAY_NOT_APPLICABLE
+
+  override def toMutable:BlueprintPair = {
+    val choice = new edu.gemini.model.p1.mutable.ZorroBlueprintChoice
+    val mBlueprint = new edu.gemini.model.p1.mutable.ZorroBlueprint
+    choice.setZorro(mBlueprint)
+
+    mBlueprint.setId(getBlueprintId)
+    mBlueprint.setName(getName)
+    mBlueprint.setMode(mode)
```
Comment (cquiroz) [inline phase1-data/src/main/java/edu/gemini/tac/persistence/phase1/blueprint/zorro/ZorroBlueprint.scala]: Do you need to set it to visitor?

**Comment by cquiroz** [comment]
Code looks good but would be good to test it

**Comment by sraaphorst** [comment]
Hmmm... Danny didn't set `AlopekeBlueprint` to visitor but it should be, as well as `ZorroBlueprint`. I'm going to make that change for both of them.

**Comment by sraaphorst** [comment]
Success achieved with the following:

[zorro.tar.gz](https://github.com/gemini-hlsw/itac/files/3055938/zorro.tar.gz)

Screen Shot 2019-04-08 at 3 22 01 PM

Screen Shot 2019-04-08 at 3 22 27 PM

**Comment by sraaphorst** [comment]
And here is the PDF of the details:

[Proposals.pdf](https://github.com/gemini-hlsw/itac/files/3055946/Proposals.pdf)

**Pull request merged**